setup.py

```diff
@@ -74,7 +74,7 @@
             "pytest",
             "pytest-cov",
             "coverage",
-            "sphinx",
+            "sphinx==5.0.2",
```

Comment:
Should we be more flexible on the micro version, e.g. `5.0.x`?

Reply:
This is tooling. It's not released as a part of the library so I prefer a fixed issue to avoid any possible breaks.

Reply:
I'm going to merge now to unblock other PRs. We can change it in the future if we change mind.